pkg/daemon/gpsd.go

```diff
@@ -280,7 +279,7 @@ retry:
 				g.sourceLost = false
 				if timeLs != nil {
 					select {
-					case g.leapManager.UbloxLsInd <- *timeLs:
+					case leap.LeapMgr.UbloxLsInd <- *timeLs:
```

Comment:
This should probably be after the event processing code (switch nStatus >= 3).  If this blocks for a second or so, then it could lead to delays writing to the event channel, which in my opinion is more important to have realtime than updating the leap file.